recipes/spdlog/all/test_package/test_package.cpp

```diff
@@ -1,59 +1,234 @@
-#include <cstdlib>
```

Comment:
Could you please remove some method from this example? I see it could be simpler. Remember, test_package is proposed for package validation only.

Reply:
This example is a very complete test, making sure a waste majority of the package is available and working.

If the test is expected to be more minimal, I have no problem shrinking this.